conf/zeppelin-env.sh.template

```diff
@@ -20,7 +20,9 @@
 # export MASTER=                 		# Spark master url. eg. spark://master_addr:7077. Leave empty if you want to use local mode.
 # export ZEPPELIN_JAVA_OPTS      		# Additional jvm options. for example, export ZEPPELIN_JAVA_OPTS="-Dspark.executor.memory=8g -Dspark.cores.max=16"
 # export ZEPPELIN_MEM            		# Zeppelin jvm mem options Default -Xmx1024m -XX:MaxPermSize=512m
-# export ZEPPELIN_INTP_MEM       		# zeppelin interpreter process jvm mem options. Default = ZEPPELIN_MEM
+# export ZEPPELIN_INTP_DEFAULT_MEM      # zeppelin interpreter process jvm mem options. Default = ZEPPELIN_MEM
+# export {INTERPRETER_ID}_INTP_MEM      # Zeppelin jvm mem options specific to intp
```

Comment:
Could you please fix the indentation of the comments? And also please remove the empty line below.